### ISSUE TITLE
File logging still occurs when deleting a submission or context

### ISSUE THREAD

**Issue by jnugent** [pkp/pkp-lib#7597]
**Describe the bug**
There is code in the PKPSubmissionFileService that logs the deletion of a submission file in the submission file log. This occurs even if the submission is deleted, or if a context is deleted.   After discussing this with @asmecher the suggestion is to:

1. Move the file deletion event to the submission log, and
2.  skipping logging entirely when deleting a submission? (This will implicitly cover the case of a journal deletion.)

**Comment by asmecher** [comment]
@jnugent, we had a little discussion about this on Slack, with @NateWr recommending that we don't skip the logging when deleting a submission (or entire journal). I think part of the initial reasoning behind your proposal was to avoid some fatal errors, which IIRC didn't turn out to be caused by logging then deleting. Were you encountering a significant performance slowdown that was traced to logging?

**Comment by jnugent** [comment]
@asmecher no, it wasn't a performance issue, it was the fact that the submission file service does this, when it logs:

'username' => Application::get()->getRequest()->getUser()->getUsername(),

and I had rolled the journal deletion code into a CLI tool and that causes the above line to Fatal Error because there's no user in the Request object when I was doing that.  Feel free to close the issue if you like in that case, I was trying to save a bit of time because I needed to delete several hundred journals from an installation and there is no way in OJS to do that quickly right now. The deletion functionality in the context grid can time out if the journal you are deleting is very large which can leave a bit of a mess behind.

I ended up commenting out the logging code completely in my situation.
